lib/openhab/dsl/items/timed_command.rb

```diff
@@ -42,6 +44,7 @@ module TimedCommand
         #   @!visibility private
         #
         TimedCommandDetails = Struct.new(:item,
+                                         :command,
```

Comment:
there should be no need to persist the command on the timed command details, just send it as a param to create_timed_command

Reply:
I was actually thinking, why not officially expose/document these things to the block. It may be useful to make a decision in choosing the reverted state based on the original command and item. WDYT?

Reply:
there's nothing in here that _should_ be available to the block, and isn't already available when the block is created

Reply:
What if the block is a proc defined elsewhere?

Reply:
I've just removed it from the Struct and passed it as an argument.